pyproject.toml

```diff
@@ -65,6 +65,9 @@ report-polarion = [
     "tmt[report-junit]",
     "tmt[export-polarion]",
     ]
+link-jira = [
+    "jira>=3.2.0",
+    ]
```

Comment:
The new optional dependency should be also created as an `rpm` package. The relevant spec file lines are here:

https://github.com/teemtee/tmt/blob/e6a0964882019f39eb3a62eade4171ef2c21e5ca/tmt.spec#L29-L31

Reply:
this was resolved